### DISCUSSION TITLE
Usage with DaisyUI?

### DISCUSSION THREAD

**Discussion by danawoodman** [maizzle/framework#827] [Q&A]
How would one go about using [DaisyUI](https://daisyui.com) with this project? 

I've included the plugin in my config but the CSS is not making it into the templates. 

Any tips would be greatly appreaciated!

**Comment by danawoodman** [comment]
On further investigation, I realized it is partially working. What the current issue is now is that Maizzle is stripping out the CSS variables and thus a lot of the shared styling breaks. Is there a way to get the variable's values inlined instead?

e.g. with:

```css
.btn {
  background-color: hsl(var(--pc));
}
```

outputs:

```css
.btn {
  background-color: hsl(0 0% 100%);
}
```

My current workaround is to manually inject the CSS properties/vars into the top of the `<style>` tag but I assume this will fail in lots of older clients and is more of a pain.

Is this possible/configurable?

Possibly related: 

- https://github.com/Automattic/juice/issues/312
- https://github.com/jakobo/codedrift/issues/103
  - https://github.com/jakobo/codedrift/discussions/104

**Comment by cossssmin** [comment]
I'd just add them in a separate style tag, email clients that [support CSS variables](https://www.caniemail.com/features/css-variables/) will also support CSS in `<style>` tags so there's really no reason to inline them. Only possible concern would be when an email is forwarded, but that's not going to work anyway, since `style` and other tags that you could use with variables would be lost anyway...

**Reply by danawoodman** [reply]
I had added the variables in a style tag but of course the support for CSS vars is really poor, so that won't be an option. I've manually converted the variables into inlined hex codes which works, though is a bit hacky unfortunately. Not sure a better approach tho
